numpy/ma/core.py

```diff
@@ -6184,17 +6184,38 @@ def isMaskedArray(x):
 
 
 class MaskedConstant(MaskedArray):
-    # We define the masked singleton as a float for higher precedence.
-    # Note that it can be tricky sometimes w/ type comparison
-    _data = data = np.array(0.)
-    _mask = mask = np.array(True)
-    _baseclass = ndarray
+
+    __singleton = None
```

Comment:
`__` to me is associated with `dunder` while this is really just a private attribute.